src/components/intern-app.html

```diff
@@ -1,75 +1,56 @@
+<link rel="import" href="./create-user.html">
+<link rel="import" href="./edit-user.html">
+<link rel="import" href="./user-list.html">
 
-<!-- Imports for icons from jha-design-components -->
-<link rel="import" href="~@banno/jha-design-components/icons/jha-icon-fingerprint.html">
-<link rel="import" href="~@banno/jha-design-components/icons/jha-icon-add.html">
-<link rel="import" href="~@banno/jha-design-components/icons/jha-icon-android.html">
-<link rel="import" href="~@banno/jha-design-components/icons/jha-icon-apple.html">
-<link rel="import" href="~@banno/jha-design-components/icons/jha-icon-assessment.html">
-<link rel="import" href="~@banno/jha-design-components/icons/jha-icon-bookmark.html">
 <dom-module id="intern-app">
   <template>
-    <!-- An example of an inline stylesheet. This does not require extra Webpack configuration --> 
-    <!-- and these styles are scoped to the element. -->
     <style>
-      :host {
-        font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
-        
-        /* Using a variable set in global.css */
-        color: var(--color-gray-base);
-      }
-      header {
-        width: 100%;
-        padding: 20px 0;
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: center;
-      }
       h1 {
-        font-weight: normal;
-      }
-      p {
-        font-weight: lighter;
-        display: inline-block;
-      }
-      span {
-        font-family: monospace;
-        font-size: 15px;
+        background-color: indigo;
+        color: #fcfcff;
+        padding: 14px 25px;
+        margin: 0px;
       }
-      #intro {
-        max-width: 600px;
-        margin: auto;
-        margin-top: 25px;
-      }
-      .icons {
-        display: flex;
-        justify-content: center;
+      h1:hover {
+        background-color: DarkSlateBlue;
       }
     </style>
     <header>
-      <!-- An example of a custom element being used in a custom component -->
-      <starter-component></starter-component>
-      <h1>You're up and running with Polymer + Webpack ??</h1>
+      <h1>Intern App</h1>
     </header>
-    <div id="intro">
-      <p>To get started, edit <span>src/components/intern-app.html</span> and save to reload.</p>
-      <div class="icons">
-        <!-- Icons from jha-design-components -->
-        <!-- Full list available at http://www.jha.design/components/icon/#icons -->
-        <jha-icon-fingerprint></jha-icon-fingerprint>
-        <jha-icon-add></jha-icon-add>
-        <jha-icon-android></jha-icon-android>
-        <jha-icon-apple></jha-icon-apple>
-        <jha-icon-assessment></jha-icon-assessment>
-        <jha-icon-bookmark></jha-icon-bookmark>
-      </div>
-    </div>
+    <user-profile 
+      id="[[user.id]]" 
+      first-name="[[user.firstName]]" 
+      last-name="[[user.lastName]]"
+      phone-number="[[user.phoneNumber]]" 
```

Comment:
While this works, getting dataBinding to know when sub-properties update can be tricky.

So if you do something to `user.firstName` from the intern-app view and it doesn't update the display in the user card, just remember that this might be the cause. 

Normally here I would just pass in the whole user object into `user-profile` and then divvy out the display by property inside of `user-profile`. It's the same end result but I've tended to have less problems updating data this way. 

```
  <user-profile user="[[user]]"></user-profile>
```

This isn't a required change, just something to keep in the back of your mind.